### PR TITLE
Dagger module for token service

### PR DESCRIPTION
**Pull request by hendrikebbers**
The token service now has a dagger module. Same work should be done for all services once this PR is approved.

### CODE REVIEW

File: hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenComponent.java [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+ */
+
+package com.hedera.node.app.service.token.impl.handlers;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * Wrapper class for all handlers of the token service
```
Comment (netopyr) [inline hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenComponent.java:24]: We should add a comment, that records are apparently not supported in Dagger and that is the reason for this class.

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandlersModule.java [hunk 2]
```diff
@@ -55,14 +55,14 @@ static TransactionHandlers provideTransactionHandlers(
                 contractComponent.contractSystemDeleteHandler(),
                 contractComponent.contractSystemUndeleteHandler(),
                 contractComponent.etherumTransactionHandler(),
-                tokenComponent.cryptoCreateHandler(),
```
Comment (netopyr) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandlersModule.java:58]: We should probably keep the old names. I see no benefit in renaming everything at this point.

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/TokenComponentTest.java
Comment (netopyr) [inline hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/TokenComponentTest.java:1]: I wonder, if we should still have a test, that checks if everything was created correctly.

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM ! As Michael H said some javadocs might not be needed

**Review by qnswirlds — APPROVED** [review]

**Comment by hendrikebbers** [comment]
Will remove Javadoc with next PR

**Pull request merged**